spec/checkNewCronJobSpec.js

```diff
@@ -0,0 +1,416 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
```

Comment:
Add a test case where the changed files include the cron test file

Reply:
done